modules/ting_search_carousel/ting_search_carousel.install

```diff
@@ -19,3 +20,37 @@ function ting_search_carousel_uninstall() {
   variable_del('ting_carousel_search_queries_hidden');
   variable_del('ting_carousel_search_queries');
 }
+
+function ting_search_carousel_add_shortcut() {
+  // Add shortcut to carousel.
+  $shortcut_set = shortcut_set_load(SHORTCUT_DEFAULT_SET_NAME);
```

Comment:
This also creates a dependency to `shortcut` which does not seem to be declared previously in this module or its dependencies.

Perhaps we should consider making it a soft dependency by wrapping it in `if (module_exists('shortcut'`) {`?